bundle/manifests/node-observability-operator-controller-role_rbac.authorization.k8s.io_v1_clusterrole.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  creationTimestamp: null
+  labels:
+    scc: node-observability-scc
+  name: node-observability-operator-controller-role
+rules:
+- apiGroups:
+  - security.openshift.io
+  resourceNames:
+  - node-observability-scc
+  resources:
+  - securitycontextconstraints
+  verbs:
+  - use
+- apiGroups:
+  - ""
+  resources:
+  - nodes
+  verbs:
+  - get
+  - list
+- apiGroups:
+  - ""
+  resources:
+  - nodes/proxy
+  verbs:
+  - get
+  - list
+- apiGroups:
+  - ""
+  resources:
+  - pods
+  verbs:
+  - get
+  - list
```

Comment:
pod permissions weren't required, but the nodes permission are needed since we make a http call to trigger debugging. 
```
time="2022-10-05T10:29:26Z" level=error msg="errors encountered while running Kubelet:\nerror with HTTP request for kubelet profiling https://10.0.158.202:10250/debug/pprof/profile: statusCode 403\n" module=handler

```

Reply:
Yes, true,[ kubelet checks the node](https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/server/auth.go#L60) permissions to allow the access to it API. Although, I doesn't seem to need the `nodes` resource, only `nodes/proxy`, `list` verb doesn't seem to be needed either.